pkg/update-readme

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+cp ../README.md .
```

Comment:
I'd just reverse that and move it to the scripts folder, what do you think?

Then it'd become that:
```suggestion
cp ./README.md ./pkg/
```

And potentially we want to run that script directly from the github workflow for releases as well to be sure we're in sync.

Reply:
Sounds good to me.